### DISCUSSION TITLE
How to prevent "regular" users from login into Zitadel

### DISCUSSION THREAD

**Discussion by cyrillSMF** [zitadel/zitadel#7697] [Q&A]
I've encountered a scenario where regular users (those who have signed up through the standard registration form) can access the Zitadel UI. This occurs when a user removes the “authRequestID” parameter from the URL and completes the login process. Although these users don't have permissions to perform actions within Zitadel, I would prefer to restrict access entirely, limiting login capabilities to Zitadel Admins only.

Has anyone addressed this issue, or is there a configuration step I might have overlooked?

Best regards,

Cyrill

**Comment by mario-shang** [comment]
Please take a look at this document to see if it can solve your problem. [Prevent users from accessing ZITADEL console](https://zitadel.com/docs/guides/solution-scenarios/restrict-console).

**Reply by cyrillSMF** [reply]
Thank you, we were now able to solve this issue.
